frontend/src/components/Forms/FormLayoutCenter/index.module.css

```diff
@@ -0,0 +1,3 @@
+.centerLayout {
+  width: 100%;
+}
```

Comment:
I don't get it. 🙂 `width: 100%` is the default.

The only thing this does is to make the input take up the entire space. But that's not `FormLayourCenter`?

Reply:
Forgot to delete this whole component that's not being used anymore. was testing 😬